src/Symfony/Component/FeatureToggle/Debug/TraceableStrategy.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureToggle\Debug;
+
+use Symfony\Bundle\FeatureToggleBundle\DataCollector\FeatureCheckerDataCollector;
```

Comment:
the component cannot depend on the bundle.

Reply:
We followed the same rules as other components for the debug part (Serializer for example)

Reply:
The TraceableSerializer does not depend on FrameworkBundle. The SerializerDataCollector is part of the component.

Reply:
Good catch, I'll move it.

Reply:
moved